src/translations/nbTranslations.json

```diff
@@ -1168,9 +1169,10 @@
         "funder": "Finansiør",
         "funding_helper_text": "<0>Du kan registrere finansiering direkte på resultatet. Særlig relevant om du ikke kjenner til hvilket prosjekt resultatet er del av, eller dersom resultatet har andre finansiører enn prosjektet det eventuelt tilhører.</0> <0>Om resultatet er tilknyttet et prosjekt vil det arve alle prosjektfinansieringer. Det er ikke nødvendig å legge inn de samme finansieringene dobbelt opp.</0>",
         "funding_sum": "Sum",
+        "funding_id": "Tilskudds ID",
         "nfr_project": "NFR-prosjekt",
         "nfr_project_search": "Søk etter prosjektnavn eller prosjektleder",
-        "project": "Prosjekt",
+        "project": "Tilskuddsnavn",
```

Comment:
Burde det ikke heller vært en key "name" for denne?

Reply:
enig, endret til funding_name